app/views/staff/proposals/_form.html.haml

```diff
@@ -10,11 +10,12 @@
       %section.inline-block
         = f.label :slides_url
         = f.text_field :slides_url, class: "form-control"
-      %p
-      = f.select :review_tags,
-        options_for_select(event.review_tags, proposal.object.review_tags),
-        {}, {class: 'multiselect review-tags', multiple: true}
-
+  .row
```

Comment:
I cleaned this up a bit, this view is getting funky due to git. But the intention here as to put the review_tags select for /events/seedconf/staff/proposals/59b6ee0a49/edit view into its own row. I took out the %p initially, but it needs the spacing to push it down. Added p tag back. The row is at the end of the edit view, so it being empty is ok.